gnat2goto/driver/range_check.adb

```diff
@@ -43,16 +47,17 @@ package body Range_Check is
    -- Load_Real_Bound_In_Hex --
    ----------------------------
 
-   function Load_Real_Bound_In_Hex (Index : Integer; Bit_Width : Integer)
+   function Load_Real_Bound_In_Hex (Index : Integer; Actual_Type : Irep)
                                     return String
    is
+      Bit_Width : constant Float_Format := To_Float_Format (Actual_Type);
       Bound : constant Ureal :=
         Ureal (Integer_Bounds_Real_Table.Element (Index));
    begin
-      return (case Bit_Width is
-                 when 32 => Convert_Ureal_To_Hex_32bits_IEEE (Bound),
-                 when 64 => Convert_Ureal_To_Hex_64bits_IEEE (Bound),
-                 when others => raise Program_Error);
-   end Load_Real_Bound_In_Hex;
+      if Bit_Width = IEEE_32_Bit then
```

Comment:
Would prefer a case statement here, as that would cause a compile time error if we add support for more float types without updating this section

Reply:
Done.